assets/components/Fetch.js

```diff
@@ -10,11 +10,14 @@ class BaseFetch extends Component {
   static propTypes = {
     children: PropTypes.func.isRequired,
     path: PropTypes.string.isRequired,
+    // eslint-disable-next-line react/forbid-prop-types
```

Comment:
If you're going to go on with PropTypes for this codebase in the forseeable future, may as well just turn off this rule in the eslint config.

Reply:
Yeah my position is I would, all else being equal, prefer TS to Proptypes, but in this particular case there's not much to be gained from a TS migration and proptypes are better than no types at all.